src/libaktualizr/package_manager/packagemanagerinterface.cc

```diff
@@ -9,7 +9,7 @@ struct DownloadMetaStruct {
         target{std::move(target_in)},
         token{token_in},
         progress_cb{std::move(progress_cb_in)} {}
-  uint64_t downloaded_length{0};
+  uintmax_t downloaded_length{0};
```

Comment:
I get why `size_t` was wrong, but is `uintmax_t` really necessary (compared to `uint64_t`)?

Reply:
I had the same question, it seems that the reason is that the c++ filesystem api (also boost) returns this type for file sizes: https://en.cppreference.com/w/cpp/filesystem/file_size.

Reply:
It should be not necessary, just wanted to be consistent with return types from boost.

Reply:
Fair enough. Seems fine.